rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -58,10 +58,10 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
 
   "evalExpr" should "handle simple addition" in {
     implicit val errorLog = new ErrorLog()
-    implicit val costAccounting =
-      CostAccountingAlg.unsafe[Task](CostAccount(Integer.MAX_VALUE))
+
     val result = withTestSpace { space =>
-      val reducer      = RholangOnlyDispatcher.create[Task, Task.Par](space)._2
+      val reducer = RholangOnlyDispatcher.create[Task, Task.Par](space)._2
+      Await.ready(reducer.setAvailablePhlos(Cost(Integer.MAX_VALUE)).runAsync, 1.second)
```

Comment:
This commit leaves no doubt as to whether we should elliminate the duplication of setup in this test first. Please do so, as proposed in review for the other PR.